Tests/IntegrationTests/App.config

```diff
@@ -8,5 +8,8 @@
         <add key="DotNetWspEntityID" value="https://wsp.oioidws-net.dk" />
         <add key="DotNetWspHostname" value="https://Digst.OioIdws.Wsp:9090/HelloWorld" />
         <add key="DotNetWspCertificatePath" value="Resources\\DotNet_WSP.p12" />
+        <add key="LocalTokenServiceSigningCertificatePath" value="Resources\\sts.oioidws-net.dk.p12" />
+        <add key="LocalTokenServiceEntityId" value="https://sts.oioidws-net.dk" />
+        <add key="NemLoginLocalTokenStsEntityId" value="https://local.sts.nemlog-in.dk/" />
```

Comment:
Denne behøves ikke at være en variabel, den er konstant for NemLogin uanset miljø.